### ISSUE TITLE
Provider statement is missing edgerc

### ISSUE THREAD

**Issue by ericrichtert** [akamai/terraform-provider-akamai#187]
After upgrading to provider v0.10.2, I get a warning and an error about the provider config:

```
Warning: Attribute is deprecated

The setting "property" has been deprecated. See:
        https://www.terraform.io/docs/configuration/providers.html#alias-multiple-provider-configurations

Error: value not found: edgerc

```
According to https://github.com/akamai/terraform-provider-akamai/blob/master/docs/index.md , the edgerc section is optional.

As I dont have a edgerc file, I need to provide the API host/tokens via a different way
### Terraform Version
Terraform v0.13.4
+ provider registry.terraform.io/akamai/akamai v0.10.2
+ provider registry.terraform.io/hashicorp/aws v3.12.0
+ provider registry.terraform.io/hashicorp/template v2.2.0

### Affected Resource(s)/Terraform Configuration Files
```hcl
provider "akamai" {
  property {
    host          = var.akamai_host
    access_token  = var.akamai_access_token
    client_token  = var.akamai_client_token
    client_secret = var.akamai_client_secret
  }
}

```

### Debug Output
```
2020-10-25T11:57:03.855+0100 [DEBUG] plugin: starting plugin: path=.terraform/plugins/registry.terraform.io/akamai/akamai/0.10.2/windows_amd64/terraform-provider-akamai_v0.10.2.exe args=[.terraform/plugins/registry.terraform.io/akamai/akamai/0.10.2/windows_amd64/terraform-provider-akamai_v0.10.2.exe]
2020/10/25 11:57:03 [TRACE] [walkRefresh] Exiting eval tree: local.rules
2020/10/25 11:57:03 [TRACE] vertex "local.rules": visit complete
2020/10/25 11:57:03 [TRACE] vertex "local.rules (expand)": dynamic subgraph completed successfully
2020/10/25 11:57:03 [TRACE] vertex "local.rules (expand)": visit complete
2020-10-25T11:57:04.020+0100 [DEBUG] plugin: plugin started: path=.terraform/plugins/registry.terraform.io/akamai/akamai/0.10.2/windows_amd64/terraform-provider-akamai_v0.10.2.exe pid=5400
2020-10-25T11:57:04.020+0100 [DEBUG] plugin: waiting for RPC address: path=.terraform/plugins/registry.terraform.io/akamai/akamai/0.10.2/windows_amd64/terraform-provider-akamai_v0.10.2.exe
2020-10-25T11:57:04.074+0100 [INFO]  plugin.terraform-provider-akamai_v0.10.2.exe: configuring server automatic mTLS: timestamp=2020-10-25T11:57:04.073+0100
2020-10-25T11:57:04.103+0100 [DEBUG] plugin: using plugin: version=5
2020-10-25T11:57:04.103+0100 [DEBUG] plugin.terraform-provider-akamai_v0.10.2.exe: plugin address: address=127.0.0.1:10000 network=tcp timestamp=2020-10-25T11:57:04.103+0100
2020-10-25T11:57:04.158+0100 [TRACE] plugin.stdio: waiting for stdio data
2020/10/25 11:57:04 [TRACE] BuiltinEvalContext: Initialized "provider[\"registry.terraform.io/akamai/akamai\"]" provider for provider["registry.terraform.io/akamai/akamai"]
2020/10/25 11:57:04 [TRACE] eval: terraform.EvalNoop
2020/10/25 11:57:04 [TRACE] eval: *terraform.EvalOpFilter
2020/10/25 11:57:04 [TRACE] eval: *terraform.EvalSequence
2020/10/25 11:57:04 [TRACE] eval: *terraform.EvalGetProvider
2020/10/25 11:57:04 [TRACE] eval: *terraform.EvalOpFilter
2020/10/25 11:57:04 [TRACE] eval: *terraform.EvalSequence
2020/10/25 11:57:04 [TRACE] eval: *terraform.EvalConfigProvider
2020/10/25 11:57:04 [TRACE] buildProviderConfig for provider["registry.terraform.io/akamai/akamai"]: using explicit config only
2020/10/25 11:57:04 [TRACE] GRPCProvider: GetSchema
2020/10/25 11:57:04 [TRACE] GRPCProvider: Configure
2020-10-25T11:57:04.163+0100 [DEBUG] plugin.terraform-provider-akamai_v0.10.2.exe: 2020-10-25T11:57:04.163+0100 [DEBUG] START Configure: OperationID=f274457e-0c65-487f-ade5-d99b257e3a0b
2020-10-25T11:57:04.163+0100 [DEBUG] plugin.terraform-provider-akamai_v0.10.2.exe: 2020-10-25T11:57:04.163+0100 [DEBUG] START Configure: OperationID=f274457e-0c65-487f-ade5-d99b257e3a0b
2020-10-25T11:57:04.163+0100 [DEBUG] plugin.terraform-provider-akamai_v0.10.2.exe: 2020-10-25T11:57:04.163+0100 [DEBUG] START Configure: OperationID=f274457e-0c65-487f-ade5-d99b257e3a0b
2020/10/25 11:57:04 [ERROR] eval: *terraform.EvalConfigProvider, err: value not found: edgerc
2020/10/25 11:57:04 [ERROR] eval: *terraform.EvalSequence, err: value not found: edgerc
2020/10/25 11:57:04 [ERROR] eval: *terraform.EvalOpFilter, err: value not found: edgerc
2020/10/25 11:57:04 [ERROR] eval: *terraform.EvalSequence, err: value not found: edgerc
2020/10/25 11:57:04 [TRACE] [walkRefresh] Exiting eval tree: provider["registry.terraform.io/akamai/akamai"]
2020/10/25 11:57:04 [TRACE] vertex "provider[\"registry.terraform.io/akamai/akamai\"]": visit complete

```
### Expected Behavior
As edgerc is optional, there should be no error

### Actual Behavior
An error

### Steps to Reproduce
1. `terraform plan`

**Comment by siwyd** [comment]
Confirmed, just wanted to upgrade from 0.9 to 0.11 and this prevents me from doing that.

**Comment by zstlaw** [comment]
Thanks for reporting this issue. It has been fixed in the 1.0.0 release. you will need to use a section named config instead of property but otherwise this appears to work in 1.0.0 release posted today.

Please try upgrading to 1.0.0 and if you still face issues, Please let us know. Please refer https://registry.terraform.io/providers/akamai/akamai/latest/docs/guides/1.0_migration for additional details.
